src/main/dynamo/Table.ts

```diff
@@ -7,19 +7,24 @@ export class Table {
   readonly tableName: string
   readonly partitionKeyName: string
   readonly sortKeyName: string
-  private encryptionBlacklist = new Set<string>(["model", "__jayz__metadata"])
+  private encryptionBlacklist: Set<string>
 
   constructor(config: {
     name: string
     partitionKeyName: string
-    sortKeyName: string
+    sortKeyName: string,
+    encryptionBlacklist?: string[]
   }) {
     this.tableName = config.name
     this.partitionKeyName = config.partitionKeyName
     this.sortKeyName = config.sortKeyName
 
-    this.addToEncryptionBlacklist(this.partitionKeyName)
-    this.addToEncryptionBlacklist(this.sortKeyName)
+    this.encryptionBlacklist = new Set([
+      "model",
+      "__jayz__metadata",
+      this.partitionKeyName,
+      this.sortKeyName,
+      ...config.encryptionBlacklist!])
```

Comment:
Think you'll need a `...(config.encryptionBlacklist || [])` here to avoid blowing up on undefined.